app/models/player.rb

```diff
@@ -1,3 +1,16 @@
 class Player < ApplicationRecord
+  include Sortable
   validates :name, presence: true, length: { maximum: 30 }
+
+  def self.allowed_sort_columns
+    %w[name]
+  end
+
+  def self.default_sort_column
+    'name'
+  end
+
+  def self.default_direction
+    'asc'
+  end
```

Comment:
The default direction is already `'asc'`, defined [here](https://github.com/robert-damoc/game-stats/blob/main/app/models/concerns/sortable.rb#L24C1-L24C1)